config.conf

```diff
@@ -10,4 +10,4 @@ KORYAVNIKS=(
 
 #PDFVIEWER
 
-pdfviewer_script=~/gnu-koryavov/okular.sh
+pdfviewer_script=~/gnu-koryavov/atril.sh
```

Comment:
I think leaving `okular` as a default pdf viewer would be better for most users. For now you can add more information about pdf viewer script selecting to the `README` if you wish